docs/deployment/cli.zh.md

```diff
@@ -334,7 +334,64 @@ parameter combinations:
 The `--target` will overwrite the [execution.target]({% link deployment/config.zh.md %}#execution-target) 
 specified in the `config/flink-config.yaml`.
 
-For more details on the commands and the available options, please refer to the Resource Provider-specific 
+For more details on the commands and the available options, please refer to the Resource Provider-specific
 pages of the documentation.
 
+### Submitting PyFlink Jobs
```

Comment:
It appears (after having only a brief look over it) that the comments I gave on the English version apply on the Chinese version as well.